README.md

```diff
@@ -237,6 +238,19 @@ overview of a manual upgrade of a Kubernetes Engine cluster.
 
 ![manual-upgrade](images/manual-upgrade.png)
 
+## Troubleshooting
+
+If Running into an error while executing `make` commands similar to the below example output
+```console
+Waiting for rollout to finish: 1 of 3 updated replicas are available...
+error: deployment "es-master" exceeded its progress deadline
+```
+The solution would be to double check if the region is set correctly and not zone.
+Run this command to set the region.
+```console
+gcloud config set compute/region us-east1
```

Comment:
we do not want to list a specific region.  Let the user decide.

Reply:
replaced default region with generic name